packages/artifact/src/internal/upload/blob-upload.ts

```diff
@@ -80,6 +109,8 @@ export async function uploadZipToBlobStorage(
     )
   }
 
+  // clear the progress timeout when upload completes
+  clearTimeout(chunkTimer(timeoutDuration))
```

Comment:
we probably don't need this anymore bc of the `finally` above

also this should use `timeoutId` not the chunk timer func